tests/apm/mod.rs

```diff
@@ -179,53 +156,52 @@ fn logging() {
     // First insert completed
     line.clear();
     read_first_non_monitor_line(&mut file, &mut line);
-    assert_start_ignoring_ip!(insert_one_line_start, line);
+    assert!(coerce_ip_v6_to_v4(&line).starts_with(insert_one_line_start));
     assert!(line.ends_with(" ns)\n"));
 
     // Second insert started
     line.clear();
     read_first_non_monitor_line(&mut file, &mut line);
-    assert_eq_ignoring_ip!(
+    assert_eq!(
         "COMMAND.insert_one 127.0.0.1:27017 STARTED: { insert: \"logging\", documents: [{ _id: 2 }] }\n",
-        line
+        coerce_ip_v6_to_v4(&line)
     );
 
     // Second insert completed
     line.clear();
     read_first_non_monitor_line(&mut file, &mut line);
-    assert_start_ignoring_ip!(insert_one_line_start, line);
+    assert!(coerce_ip_v6_to_v4(&line).starts_with(insert_one_line_start));
     assert!(line.ends_with(" ns)\n"));
 
     // Third insert started
     line.clear();
     read_first_non_monitor_line(&mut file, &mut line);
-    assert_start_ignoring_ip!(
+    assert!(
         "COMMAND.insert_one 127.0.0.1:27017 STARTED: { insert: \"logging\", documents: [{ _id: 3 }] }\n",
-        line
+        coerce_ip_v6_to_v4(&line)
```

Comment:
This one needs to be `assert!(coerce(line).starts_with())`

Reply:
Fixes